yaml/pom.xml

```diff
@@ -0,0 +1,126 @@
+  <modelVersion>4.0.0</modelVersion> 
+  <parent>
+    <groupId>com.fasterxml.jackson.jaxrs</groupId>
+    <artifactId>jackson-jaxrs-providers</artifactId>
+    <version>2.6.0-rc2-SNAPSHOT</version>
+  </parent>
+  <artifactId>jackson-jaxrs-yaml-provider</artifactId>
+  <name>Jackson-JAXRS-YAML</name>
+  <packaging>bundle</packaging>
+  <description>Functionality to handle YAML input/output for JAX-RS implementations (like Jersey and RESTeasy) using standard Jackson data binding.
+  </description>
+
+  <properties>
+    <!-- Generate PackageVersion.java into this directory. -->
+    <packageVersion.dir>com/fasterxml/jackson/jaxrs/yaml</packageVersion.dir>
+    <packageVersion.package>${project.groupId}.yaml</packageVersion.package>
+    <osgi.export>${project.groupId}.yaml.*;version=${project.version}</osgi.export>
+    <!-- NOTE: JAXB annotations module is optional dependency, need to try to mark
+         as such here.
+      -->
+    <osgi.import>javax.ws.rs;version="${javax.ws.rs.version}"
+,javax.ws.rs.core;version="${javax.ws.rs.version}"
+,javax.ws.rs.ext;version="${javax.ws.rs.version}"
+,com.fasterxml.jackson.annotation
+,com.fasterxml.jackson.core
+,com.fasterxml.jackson.core.type
+,com.fasterxml.jackson.core.util
+,com.fasterxml.jackson.databind
+,com.fasterxml.jackson.databind.introspect
+,com.fasterxml.jackson.databind.type
+,com.fasterxml.jackson.databind.util
+,com.fasterxml.jackson.dataformat.yaml
+,com.fasterxml.jackson.jaxrs.base
+,com.fasterxml.jackson.jaxrs.cfg
+,com.fasterxml.jackson.module.jaxb;resolution:=optional
+</osgi.import>
+  </properties>
+
+  <dependencies>
+    <!-- builds on shared base JAX-RS handling code... -->
+    <dependency>
+      <artifactId>jackson-jaxrs-base</artifactId>
+      <groupId>${project.groupId}</groupId>
+      <version>${project.version}</version>
+    </dependency>
+    <!-- Extends Jackson core, mapper, and also (sort of optionally) on JAXB annotation handler -->
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-core</artifactId>
+      <version>${version.jackson.core}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>${version.jackson.core}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.dataformat</groupId>
+      <artifactId>jackson-dataformat-yaml</artifactId>
+      <version>${version.jackson.yaml}</version>
+    </dependency>
+    <!-- may also need JAXB annotation support -->
+    <dependency> 
+      <groupId>com.fasterxml.jackson.module</groupId>
+      <artifactId>jackson-module-jaxb-annotations</artifactId>
+      <version>${version.jackson.jaxb}</version>
+    </dependency>
+    <!--  But Stax2 API must be included -->
+    <dependency>
```

Comment:
Woodstox and stax2 only needed by XML module, so these can be removed. JAXB annotation support does not need them.